assets/src/scss/base/_mixins.scss

```diff
@@ -40,6 +40,27 @@
   }
 }
 
+// Mobile and small width (less than 768px)
+@mixin small-and-less {
```

Comment:
Since we use a mobile first approach, our code is structured by first including the styles for the smallest screens, then use `min-width` queries. 

Is there any code here that really needs a `max-width` query and can not be rewritten to also use a mobile first approach? Maybe the CSS targeting the mobile touch menu?

Reply:
I ended up using this because the min-width approach forces to overwrite some properties that are actually useless in other widths, especially for something like the menu which has 2 very different styles, and 2 sub-styles for each. I'm gonna give it a shot at mobile first, see if I can remove those mixins.

Reply:
I removed small-and-less and large-and-less :+1: . For medium-and-less, this allows to have no side effect when you stretch your window from a medium width with menu open to larger width - the alternatives seems a lot more complex to me.